sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1343,6 +1381,7 @@ def __init__(
     self._controller = controller
     self._get_buffer = get_buffer
     self._get_input_coder_impl = get_input_coder_impl
+    self._num_workers = num_workers
```

Comment:
This argument can now be omitted.

Reply:
This is used at L:1540. Default num_workers is got from self._num_workers, and it can be overwritten if num_worker is passed to `process_bundle`.

Reply:
I see. We should capture this in the constructor of ParallelBundleProcessor (which may call its superclass constructor), not here.